addons/sale/res_config_view.xml

```diff
@@ -70,6 +82,25 @@
                         </div>
                     </group>
                 </group>
+                <group name="Sale" position="after">
+                    <group name="E-Commerce Features">
+                        <label for="id" string="E-Commerce Features"/>
```

Comment:
Why adding a label ? `string` attribute on `group` tag did not do the job ?
`name` attribute above is not made for that : it should contain an 'identifier' in underscore notation to apply xpath in an easy way.